.travis.yml

```diff
@@ -90,10 +90,16 @@ matrix:
         - CPPFLAGS=-DXXH_VECTOR=4 CFLAGS="-O3 -maltivec -mvsx -mpower8-vector -mcpu=power8" LDFLAGS="-static" make check
 
     - name: IBM s390x compilation and consistency checks
+      dist: bionic
```

Comment:
Is there a specific reason to pin `dist` to `bionic` ?

Reply:
xenial only has GCC 5.4, which doesn't support s390x vector.